### PR TITLE
Debug mongoose queries and Add angular-translate

### PR DESCRIPTION
**Pull request by rubenjgarcia**
Option to debug mongoose queries on development environment

### CODE REVIEW

**Comment by ilanbiala** [comment]
@rubenjgarcia can you split this PR into 2?

File: config/env/development.js [hunk 1]
```diff
@@ -39,5 +39,8 @@ module.exports = {
 				pass: process.env.MAILER_PASSWORD || 'MAILER_PASSWORD'
 			}
 		}
-	}
+	},
+    mongoose: {
+        debug: true
```
Comment (ilanbiala) [inline config/env/development.js]: I think we should make it `db: {}` and put the db key with the url under this object as url.

**Comment by ilanbiala** [comment]
@rubenjgarcia can you pull out the mongoose debugging into a separate PR? We'll merge that into 0.4.0 because it's a good idea.

**Comment by codydaig** [comment]
Since @lirantal took the mongoose debug features from here and did it in #707, can this be closed?

**Comment by lirantal** [comment]
yeah, thanks @codydaig

**Pull request closed without merging**